src/cli/pack_command.zig

```diff
@@ -2622,12 +2622,19 @@ pub const bindings = struct {
                     return global.throw("failed to read archive header: {s}", .{Archive.errorString(@ptrCast(archive))});
                 },
                 else => {
-                    const pathname = archive_entry.pathname();
+                    const pathname_string = if (bun.Environment.isWindows) blk: {
+                        const pathname_w = archive_entry.pathnameW();
+                        const list = std.ArrayList(u8).init(bun.default_allocator);
+                        var result = bun.strings.toUTF8ListWithType(list, []const u16, pathname_w) catch bun.outOfMemory();
+                        defer result.deinit();
+                        break :blk String.cloneUTF8(result.items);
+                    } else String.cloneUTF8(archive_entry.pathname());
+
```

Comment:
```suggestion
                    const pathname_string = String.cloneUTF8(archive_entry.pathnameUtf8());

```

This change is a little weird because no tests fail if you revert it, but it does look like a bug because we'd possibly recieve non-utf8

Reply:
actually don't use pathnameUtf8. libarchive fails to convert to utf8 in our tests. I'd keep this the same